server/runtime_status.go

```diff
@@ -15,31 +19,26 @@ func (s *Server) Status(ctx context.Context, req *pb.StatusRequest) (resp *pb.St
 		recordError(operation, err)
 	}()
 
-	// Deal with Runtime conditions
-	runtimeReady, err := s.Runtime().RuntimeReady()
-	if err != nil {
-		return nil, err
+	runtimeCondition := &pb.RuntimeCondition{
+		Type:   pb.RuntimeReady,
+		Status: true,
```

Comment:
We were hard-coding `true` here before, so what you have here doesn't make this worse.  But it would be nice if we actually checked to see how the runtime was doing instead of always setting `true`.  Do we want to handle that in this PR, or punt for now?